support-files/mariadb.service

```diff
@@ -0,0 +1,104 @@
+#
+#  /etc/systemd/system/mariadb.service
+#
+
+[Unit]
+Description=MariaDB database server
+After=network.target
+After=syslog.target
```

Comment:
While it can, I think people generally expect it to be there and this mitigates a lot of the dbms shouldn't use socket activation because they are slow to start arguments.